draft-packages/illustration/docs/IllustrationScene.stories.tsx

```diff
@@ -133,7 +133,7 @@ const IllustrationScenesTemplate: StoryFn<IllustrationScenesTemplateProps> = ({
   sceneComponents,
   ...restArgs
 }: IllustrationScenesTemplateProps) => {
-  const { isAnimated, loop, autoplay, alt = "", ...restProps } = restArgs
+  const { isAnimated, loop, autoplay, alt: _, ...restProps } = restArgs
```

Comment:
```suggestion
  const { isAnimated, loop, autoplay, ...restProps } = restArgs
```

We want to keep it as part of `restProps` right?

Reply:
The problem is that there's a discriminated union happening, animated ones don't take alt text. So if I don't pull this out, I end up with a TS error below because it ends up getting added to the animated example.